src/CircleEditor/MessageType.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Custom message types to communicate with vscode and webviews
```

Comment:
Overall comment is good :)
However, I am not sure for the purpose of each message types :(
Could you write down at each message types about the details? (if possible, with example)

Reply:
ResponseModel is type for send circle model data.

RequestMessage contains data user want to edit.

ResponseModelPath is sent before send model data. It is send circle file path.

ResponseFileRequest is to send infomation about file in media folder. we use it when webview request media/CircleEditor/circle-metatdata.json

ResponseData is used when we want to send any type of data. we will use it to send json model string data and CustomOperatorInfo.

Reply:
Ah... I was to ask you to write it as comment :sweat_smile:

Reply:
For example,

```ts
// Data which user want to edit
//
// Example (Sending sample command)
//   - command : 'sample command'
//   - type : 'sample type'
//   - data : 'some data'
export interface RequestMessage {
  command: string;
  type: string;
  data: any;
}
```

Reply:
We will rearrange comments and commit again. Thank you.